component/site/libraries/commonfunctions.php

```diff
@@ -380,8 +380,12 @@ public static function sendAdminMail( $adminName, $adminEmail, $subject='', $tit
 		$messagetemplate = str_replace("{TITLE}", $title,$messagetemplate);
 		$messagetemplate = str_replace("{DESCRIPTION}", $content,$messagetemplate);
 		if ($event){
+			$location = $event->location();
+			if (isset($event->_loc_title)){ $location = $event->_loc_title(); }
```

Comment:
There's no such function - it should be 
`if (isset($event->_loc_title)){ $location = $event->_loc_title; }`